src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/ioc/ResumePaymentCallbackHandlerTest.java

```diff
@@ -121,6 +125,11 @@ public void shouldCreatePaymentIfPaymentIsFailed()
             entry("paymentDateCreated", "2017-12-03"),
             entry("paymentNextUrl", NEXT_URL)
         );
+        List<Map<String, Object>> respondents = (List<Map<String, Object>>) responseData.get("respondents");
+        Map<String, Object> defendant = (Map<String, Object>) respondents.get(0).get("value");
+
+        assertThat(defendant).contains(entry("servedDate", ISSUE_DATE.plusDays(5).toString()));
```

Comment:
Should this be serviceDate rather than servedDate

Reply:
here we are comparing against ccd data element which is served date in respondent.